src/views/Farms/components/FarmCard/FarmCard.tsx

```diff
@@ -133,9 +134,25 @@ const FarmCard: React.FC<FarmCardProps> = ({ farm, removed, cakePrice, bnbPrice,
         tokenSymbol={farm.tokenSymbol}
       />
       {!removed && (
-        <Flex justifyContent="space-between">
+        <Flex justifyContent="space-between" alignItems="center">
           <Text>{TranslateString(352, 'APY')}:</Text>
-          <Text bold>{farm.apy ? `${farmAPY}%` : 'Loading ...'}</Text>
+          <Text bold style={{ display: 'flex', alignItems: 'center' }}>
+            {farm.apy ? (
+              <>
+                ${farmAPY}%
+                <ApyButton
+                  lpLabel={lpLabel}
+                  quoteTokenAdresses={quoteTokenAdresses}
+                  quoteTokenSymbol={quoteTokenSymbol}
+                  tokenAddresses={tokenAddresses}
+                  cakePrice={cakePrice}
+                  apy={farm.apy}
+                />
+              </>
+            ) : (
+              'Loading ...'
```

Comment:
Can we use the `<Skeleton />` component here instead.

Reply:
Done 👍